target/linux/bmips/image/bcm63xx.mk

```diff
@@ -0,0 +1,33 @@
+#
+# BCM63xx Profiles
+#
+
+define Device/bcm63xx_cfe
+  FILESYSTEMS := squashfs jffs2-64k jffs2-128k
+  KERNEL := kernel-bin | append-dtb | relocate-kernel | lzma-bcm63xx | lzma-cfe
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | loader-lzma elf
+  KERNEL_INITRAMFS_SUFFIX := .elf
+  IMAGES := cfe.bin sysupgrade.bin
+  IMAGE/cfe.bin := cfe-bin --pad $$$$(shell expr $$$$(FLASH_MB) / 2)
+  IMAGE/sysupgrade.bin := cfe-bin
+  BLOCK_SIZE := 0x10000
+  IMAGE_OFFSET :=
+  FLASH_MB := 4
```

Comment:
I do not think it's a good idea to provide a default for that value. From my experience at other targets, variables like this are frequently overlooked, and then just people set nothing and inherit the 4 MiB for a device with different size (don't know whether that's technically possible here).

So, I'd put the `FLASH_MB :=` initializer here and then require to set the actual value  for _every_ device.